doc/source/examples/JobAnalysis/job_analysis.ipynb.rst

```diff
@@ -124,94 +124,102 @@ Jobs can be loaded by passing job objects directly, or loading from a path.
        ja.remove_field("Path")
        .add_standard_fields(["Formula", "Smiles", "CPUTime", "SysTime"])
+       .add_rkf_field("General", "engine")
        .add_settings_input_fields()
        .add_field("Energy", lambda j: j.results.get_energy(unit="kJ/mol"), display_name="Energy [kJ/mol]", fmt=".2f")
-       .display_table(max_rows=5)
    )
-
-======= ===== ===== ================================= ================= ================= ========= ======== ==================== ================= ============= ================= ===================== ===============
-Name    OK    Check ErrorMsg                          Formula           Smiles            CPUTime   SysTime  InputAmsTask         InputAdfBasisType InputAdfXcGga InputAmsNebImages InputAmsNebIterations Energy [kJ/mol]
-======= ===== ===== ================================= ================= ================= ========= ======== ==================== ================= ============= ================= ===================== ===============
-dftb    True  True  None                              C2H6              CC                0.210759  0.022143 SinglePoint          None              None          None              None                  -19594.01
-adf     True  True  None                              C2H6              CC                4.274061  0.271191 SinglePoint          DZ                None          None              None                  -3973.29
-…       …     …     …                                 …                 …                 …         …        …                    …                 …             …                 …                     …
-adf.016 True  True  None                              CH4O              CO                20.258706 1.389218 GeometryOptimization TZP               PBE           None              None                  -2900.38
-neb     False False NEB optimization did NOT converge : CHN, final: CHN : C=N, final: C#N 0.523557  0.087520 NEB                  None              None          9                 10                    None
-neb.002 True  True  None                              : CHN, final: CHN : C=N, final: C#N 0.869299  0.166113 NEB                  None              None          9                 100                   -14936.53
-======= ===== ===== ================================= ================= ================= ========= ======== ==================== ================= ============= ================= ===================== ===============
+   ja.display_table(max_rows=5)
+
+======= ===== ===== ================================= ================= ================= ========= ======== ================== ==================== ================= ============= ================= ===================== ===============
+Name    OK    Check ErrorMsg                          Formula           Smiles            CPUTime   SysTime  ams:General%engine InputAmsTask         InputAdfBasisType InputAdfXcGga InputAmsNebImages InputAmsNebIterations Energy [kJ/mol]
```

Comment:
e.g. this is annoying to compare, is the change because of a bug in the "return a copy" or because of the new feature?

Reply:
The new feature, next time I will comment the appropriate section also. New column is `ams:General%engine` i.e. from `.add_rkf_field("General", "engine")`

Reply:
Just to be clear, I used this format before as a quick hack, not sure this is the best name for the column (maybe it is?)